plugins/org.eclipse.fordiac.ide.systemmanagement/src/org/eclipse/fordiac/ide/systemmanagement/changelistener/FordiacResourceChangeListener.java

```diff
@@ -218,21 +164,11 @@ private void handleFileDelete(final IResourceDelta delta) {
 		final TypeEntry entry = typeLib.getTypeEntry(file);
 		if (null != entry) {
 			closeAllEditorsForFile(file);
-			final FileToRenameEntry rnEntry = getFileRenameEntry(entry);
 			typeLib.removeTypeEntry(entry);
-			if (rnEntry != null) {
-				entry.setFile(rnEntry.getFile());
-				typeLib.addTypeEntry(entry);
-				filesToRename.remove(rnEntry);
-			}
 		}
 	}
 
-	private FileToRenameEntry getFileRenameEntry(final TypeEntry palEntry) {
-		return filesToRename.stream().filter(entry -> entry.getTypeEntry().equals(palEntry)).findAny().orElse(null);
-	}
-
-	private void handleFileCopy(final IResourceDelta delta) {
+	private static void handleFileCopy(final IResourceDelta delta) {
```

Comment:
I would suggest also renaming this to match `handleResourceAdded`:
```suggestion
	private static void handleFileAdded(final IResourceDelta delta) {
```